src/main/java/org/minions/devfund/bruno/html/HtmlValidator.java

```diff
@@ -0,0 +1,28 @@
+package org.minions.devfund.bruno.html;
+
+import java.util.Queue;
+import java.util.Stack;
+
+public class HtmlValidator {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)